enterprise_access/apps/subsidy_access_policy/models.py

```diff
@@ -950,7 +951,15 @@ def can_allocate(self, number_of_learners, content_key, content_price_cents):
         into account to determine if ``number_of_learners`` new assignment
         records can be allocated in this policy for the given ``content_key``
         and it's current ``content_price_cents``.
+
+        Params:
+          number_of_learners: Non-negative integer indicating the number of learners to allocate this content to.
+          content_key: Typically a course key (although theoretically could be *any* content identifier).
+          content_price_cents: A *negative* integer reflecting the current price of the content in USD cents.
         """
+        if content_price_cents > 0:
```

Comment:
[curious/clarification] Why is `content_price_cents` expected to be negative here when the `ContentMetadataViewSet` in enterprise-subsidy where we're sourcing the price from returns content price as a positive? Should the sign (positive/negative) of content metadata be consistent throughout?

```json
{
    "content_uuid": "0912928c-00cb-44ca-87a0-e0eb75584267",
    "content_key": "OxfordX+OSMEPS",
    "course_run_uuid": "3477b85d-0b85-449c-a211-84fce307b994",
    "course_run_key": "course-v1:OxfordX+OSMEPS+1T2023",
    "source": "2u",
    "mode": "paid-executive-education",
    "content_price": 726100,
    "geag_variant_id": "bd72ac93-e11b-4707-b4b4-772b3c660abe"
}
```

Reply:
Naive answer to this question: I changed it to explicitly expect a negative number here to be consistent with what the allocate view (written like 2 weeks ago) currently expects: a negative number in cents.
Real answer to this question: we should do whatever is most reasonable and obvious from the perspective of a client of the allocate view.  If the most reasonable thing is to pass in whatever price is in the metadata (positive cents), then we can do that.